contracts/core/NPassCore.sol

```diff
@@ -24,6 +24,8 @@ abstract contract NPassCore is ERC721Enumerable, ReentrancyGuard, Ownable {
     uint256 public immutable priceForNHoldersInWei;
     uint256 public immutable priceForOpenMintInWei;
 
+    event Mint(address to, uint256 tokenId);
```

Comment:
Past tense is better for events (as they have already happened)

Reply:
Open Zeppelin uses present tense, but past tense makes sense.

Reply:
Yep, that's common in Solidity but I am fighting a war to follow 10 years of Event Sourcing conventions